dist/tools/pyterm/pyterm

```diff
@@ -106,6 +106,8 @@ class SerCmd(cmd.Cmd):
         self.run_name = run_name
         self.log_dir_name = log_dir_name
         self.newline = newline
+        if formatter:
+            self.fmt_str = formatter
```

Comment:
(`if formatter` also true for `formatter == ""` or `formatter == 0` etc, so this kind of `if ... else ...` is still sensible)